internal/runner/verify.go

```diff
@@ -46,6 +46,10 @@ func (r *VerifyRunner) Run(ctx context.Context) ([]output.CheckResult, error) {
 
 	var results []output.CheckResult
 	for result := range ch {
+		if result.Error != nil {
```

Comment:
Should we wrap all errors? E.g. in case of multiple tests with compilation errors it would be nice to report all of them.

Reply:
I think there's value in returning all of them, but it would require a bit more handling to get them all to propagate. I went with this approach because it was really straight forward, and I imagine that _most_ of the time these errors should be caught by linters. The most important thing to me in this case is that we don't say it passed.

Maybe we go with a single error to start and see if there's a high demand for needing all of them?

Reply:
Sounds like a reasonable approach for now!